src/ProjectModule/Components/Home/Home.tsx

```diff
@@ -1,7 +1,185 @@
+import styles from "./Home.module.css"
+import { Doughnut } from 'react-chartjs-2';
+import axios from 'axios';
+import {Tooltip,Title,ArcElement,Legend,Chart as ChartJS} from "chart.js"
+import { baseUrl } from '../../../Constants/Components/Urls';
+import { AuthContext } from '../../../Context/Components/AuthContext';
+ChartJS.register(Tooltip,Title,ArcElement,Legend);
+
 
 export default function Home() {
+const[taskToDo,setTaskToDo]=useState(0);
+const[taskProgres,setTaskProgres]=useState(0);
+const[taskDone,setTaskDone]=useState(0);
+const[userActive,setUserActive]=useState(0);
+const[userInActive,setUserInActive]=useState(0);
+const{userRole}=useContext(AuthContext)
+
+
+const getTasksCount=async()=>{
+  try {
+    const token=localStorage.getItem("userToken")
+    const response=await axios.get(`${baseUrl}/Task/count`,{headers:{Authorization:token}}) 
+    setTaskToDo(response.data.toDo);
+    setTaskProgres(response.data.inProgress);
+    setTaskDone(response.data.done);
+  
+    console.log(response?.data);
+   
+  } catch (error) {
+    console.log(error)
+  }
+}
+
+
+const getUsersCount=async()=>{
+  try {
+    const token=localStorage.getItem("userToken")
+    const response=await axios.get(`${baseUrl}/Users/count`,{headers:{Authorization:token}}) 
+    setUserActive(response.data.activatedEmployeeCount);
+    setUserInActive(response.data.deactivatedEmployeeCount);
+   
+    console.log(response?.data);
+  } catch (error) {
+    console.log(error)
+  }
+}
+const dataProgress = {
+  datasets: [{
+      data: [taskToDo,taskProgres,taskDone],
+      backgroundColor:[
+        " rgb(198, 160, 182)",
+       'rgb(67, 135, 64)',
+       " rgb(168, 176, 249)"
+      ],
+  }],
+
+  // These labels appear in the legend and in the tooltips when hovering different arcs
+  labels: [
+      'ToDo',
+      "inProgress",
+      "Done"
+  ]
+};
+
+const dataActive = {
+  datasets: [{
+      data: [userActive,userInActive],
+      backgroundColor:[
+        " rgb(114, 154, 145)",
+       ' rgb(217, 138, 147)',
+      
+      ],
+  }],
+
+  // These labels appear in the legend and in the tooltips when hovering different arcs
+  labels: [
+      "Active",
+      "InActive"
+  ]
+};
+
+useEffect(()=>{
+  getTasksCount();
+  getUsersCount();
+ 
+},[])
+
   return (
-    <div>Home</div>
+    <>
+    <div className="container-fluid  pt-3">
+      <div className="row ">
+        <div className="col-md-6 ">
+<div className="row gx-0">
+  <div>
+  <div className='d-flex '>
+      <div className={`${styles.vertical}`}></div>
+      <div className='p-3'><h3>Tasks</h3>
+      <p>Lorem fuga molestias, velit eius neque doloribus numquam facere vel </p></div>
+        
+      </div>
+  </div>
+  <div className="col-md-4">
+  <div className={` ${styles.designTask} rounded-4 pt-2 px-3 `} style={{backgroundColor:"#d5d7e7"}}>
+
+<i className={`fa-brands fa-squarespace  rounded-4 p-2 fa-2x mx-2 my-2`} style={{backgroundColor:"#ced1f5"}}></i>
```

Comment:
📝 Add alt attributes to `<i>` elements with `FontAwesome` icons for accessibility.